nipype/interfaces/spm/preprocess.py

```diff
@@ -177,6 +177,10 @@ class RealignOutputSpec(TraitedSpec):
                                              desc=('Estimated translation and '
                                                    'rotation parameters'))
 
+    header_files = OutputMultiPath(traits.Either(
```

Comment:
for spm, let's just call these `estimated_transform_files`, as these would be true for coregister as well when using 4d files.